src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/scalable/TruthSensitivityTranche.java

```diff
@@ -0,0 +1,386 @@
+package org.broadinstitute.hellbender.tools.walkers.vqsr.scalable;
```

Comment:
"Tranche" is a really arcane word.  Can we break with legacy GATK and just call them thresholds?

Reply:
Yes, all of this code is really arcane! I did some cleaning up here, but still a lot of vestigial weirdness left. I think even just calling these quantiles or something like that would be more intuitive. But all of this tranche stuff and how much functionality needs to be retained is probably one of the biggest open design questions.